app/src/main/java/com/compscieddy/meetinthemiddle/util/Util.java

```diff
@@ -63,4 +68,15 @@ public static void rotateLocationInactive(View view) {
         .start();
   }
 
+  public static Date getCurrentUTCTime(){
+    // Set device's time zone to UTC
+    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    Calendar calendar = Calendar.getInstance();
+    // Reset device's time zone to device local time zone
+    return calendar.getTime();
+  }
+
+  public static Date getLocalTimeFromUTC(Date utcDate){
+   return new Date(utcDate.getTime() + TimeZone.getDefault().getOffset(utcDate.getTime()));
```

Comment:
Can you add a 1-line comment explaining what you're doing. It's like almost simple enough to understand, but timezones can be a confusing concept 😃